Dockerfile

```diff
@@ -20,7 +20,7 @@ ARG	GRAFANA_ENV
 
 COPY	./grafana/${GRAFANA_ENV}/	/grafana
 
-FROM	grafana/grafana-oss:10.3.4	AS	grafana
+FROM	grafana/grafana-oss:10.4.2	AS	grafana
```

Comment:
This version introduced this bug https://github.com/grafana/grafana/issues/86505
Refreshing a grafana page helps so not critical